### DISCUSSION TITLE
Route Timetable

### DISCUSSION THREAD

**Discussion by LynxChoi** [voxeltycoon/issues#904] [General]
HI Everyone
Does any Route Timetable function to separate the space between vehicles at same route, to avoid jams and empty loads?

Thanks for any information

**Comment by SharkROAR** [comment]
There should be an option to automatically space out vehicles on a line.  This is especially necessary for passenger routes, since the "cargo" at station expires.  It doesn't help if I have 4 vehicles come in simultaneously and then have nothing come in for a long time.
